cmd/kubectl-k8ssandra/register/register_test.go

```diff
@@ -0,0 +1,178 @@
+package register
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+	"time"
+
+	configapi "github.com/k8ssandra/k8ssandra-operator/apis/config/v1beta1"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestRegister(t *testing.T) {
+	require.New(t)
```

Comment:
question: What does this do? Why not ``require := require.New(t)`` so you don't have to replicate the t on every require call?

Reply:
Ah, I think that was indeed what I meant... Thank you.